From b7d3dba6b024dc292274f1dabed8376c8abfbf49 Mon Sep 17 00:00:00 2001
From: Savelii Motov <s.motov@samsung.com>
Date: Fri, 12 May 2023 16:30:02 +0300
Subject: [PATCH 120/135] [cmake] Rename exporting libs

- Rename libAxdimmAI.so to LibPnmAI.so
- Rename libAxdimmSecure.so to LibPnmAISecure.so
- Rename COMMON_LIBS in benchmarks to BENCHMARK_LIBS

Signed-off-by: Savelii Motov <s.motov@samsung.com>
---
 caffe2/CMakeLists.txt    | 2 +-
 cmake/Dependencies.cmake | 3 +--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/caffe2/CMakeLists.txt b/caffe2/CMakeLists.txt
index 40c37153..03ba060d 100644
--- a/caffe2/CMakeLists.txt
+++ b/caffe2/CMakeLists.txt
@@ -1998,7 +1998,7 @@ if(BUILD_PYTHON)
     target_include_directories(caffe2_pybind11_state_pnm PRIVATE $<INSTALL_INTERFACE:include>)
     target_include_directories(caffe2_pybind11_state_pnm PRIVATE ${Caffe2_CPU_INCLUDE} ${Caffe2_PNM_INCLUDE})
     target_link_libraries(caffe2_pybind11_state_pnm PRIVATE torch_library python::python pybind::pybind11)
-    target_link_libraries(caffe2_pybind11_state_pnm PRIVATE PNM::AxdimmAI)
+    target_link_libraries(caffe2_pybind11_state_pnm PRIVATE PNM::PnmAI)
 
     if(USE_MKLDNN)
         target_link_libraries(caffe2_pybind11_state_pnm PRIVATE caffe2::mkldnn)
diff --git a/cmake/Dependencies.cmake b/cmake/Dependencies.cmake
index a1e8b5b8..e7faebd3 100644
--- a/cmake/Dependencies.cmake
+++ b/cmake/Dependencies.cmake
@@ -1831,8 +1831,7 @@ if(USE_PNM)
   list(APPEND CMAKE_PREFIX_PATH ${PNM_INSTALL_DIR})
   find_package(PNMLib REQUIRED)
 
-  # [TODO:] Rename to PNM* after axdimm_driver rebranding.
-  set(PNM_LIBS PNM::AxdimmSecure PNM::AxdimmAI)
+  set(PNM_LIBS PNM::PnmAISecure PNM::PnmAI)
 
   list(APPEND Caffe2_DEPENDENCY_LIBS ${PNM_LIBS})
   list(APPEND Caffe2_PNM_DEPENDENCY_LIBS ${PNM_LIBS})
-- 
2.34.1

